recipe/meta.yaml

```diff
@@ -63,6 +64,36 @@ test:
   imports:
     - astropy
 
+outputs:
+  - name: astropy-base
+
+{% if astropy_outputs == "all" %}
+  - name: astropy
+    build:
+      noarch: generic
+    requirements:
+      host:
+        - python
+      run:
+        - python
+        - {{ pin_subpackage('astropy-base', max_pin="x.x.x") }}
+        - matplotlib-base >=3.5.0,!=3.5.2
+        - certifi
+        - dask[array] >=2022.8.1
+        - h5py
+        - pyarrow >=10.0.1
+        - beautifulsoup4
+        - html5lib
+        - bleach
+        - pandas >=2.0
+        - sortedcontainers
+        - pytz
+        - jplephem
+        - mpmath
+        - bottleneck
+        - fsspec[http] >=2023.4.0
```

Comment:
the [matchspec](https://docs.conda.io/projects/conda/en/latest/dev-guide/api/conda/models/match_spec/index.html) syntax doesn't support `[whatever]` the same way as PEP508... https://github.com/conda-forge/filesystem-spec-feedstock/issues/60